aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -21,6 +21,35 @@ An ASP.NET Core app can be hosted on Windows as a [Windows Service](/dotnet/fram
 * [ASP.NET Core SDK 2.1 or later](https://dotnet.microsoft.com/download)
 * [PowerShell 6.2 or later](https://github.com/PowerShell/PowerShell)
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Worker Service template
```

Comment:
@guardrex  if this is duplicated content, should I move it to an include?

Reply:
IMO, **_not just yet_** ... not until I get to https://github.com/aspnet/AspNetCore.Docs/issues/12841. IIRC, the light coverage on this template was provided just to surface its existence at the time. I have a feeling that that these two topics (background tasks and WS) aren't going to say the same thing about the use of the template, so an INCLUDE might not work out. Not sure ... I need to get to that issue and analyze the coverage and see what's what.